eng/pipelines/official.yml

```diff
@@ -140,177 +113,120 @@ parameters:
-    dependsOn:
-    # The Build dependsOn is required for putting that stage's variables into the stageDependencies property bag (for PackageVersion).
-    - Build
-    - Publish
-    # Only include the Compliance stage when it is not skipped.
-    - ${{ if eq(parameters.SkipCompliance, false) }}:
-      - Compliance
-    variables:
-      # Gets the PackageVersion variable produced by the Build pipeline.
-      PackageVersion: $[ stageDependencies.Build.BuildOfficialRelease.outputs['SetPackageVersion.PackageVersion'] ]
-      # Gets the AssemblyVersion variable produced by the Build pipeline.
-      AssemblyVersion: $[ stageDependencies.Build.BuildOfficialRelease.outputs['SetAssemblyVersion.AssemblyVersion'] ]
-      InsertionVSBranch: ${{ parameters.InsertionVSBranch }}
-      # Hard-coded assumption that the commit referenced by the previous VS insertion is within the last 100 commits in the $(InsertionVSBranch) of the VS repo.
-      PriorInsertionCommitDepth: 100
-      # This variable is required by the component-insertion.yml template to insert the NPM package.
-      # The name of this package matches name defined in the package.json.
-      InsertNpmDependencies: '@microsoft/microsoft.visualstudio.projectsystem.managed@$(PackageVersion)'
-      InsertReviewers: '.NET Project System'
-    jobs:
-    - ${{ if or(eq(parameters.CreateVSInsertion, true), eq(variables['Build.Reason'], 'Schedule')) }}:    
-      - template: templates/generate-vs-insertion.yml
-
-    - ${{ if or(eq(parameters.CreateVSGreenInsertion, true), eq(variables['Build.Reason'], 'Schedule')) }}:
-      - template: azure-pipelines/external/component-insertion.yml@VSCodeExtensionRepo
-        parameters:
-          sourceBranch: $(Build.SourceBranch)
-          sourceCommit: $(Build.SourceVersion)
-          runName: $(Build.BuildNumber)
-          runID: $(Build.BuildId)
-          pipelineID: $(System.DefinitionId)
-          InsertTargetBranch: ${{ parameters.InsertionVSGreenBranch }}
-          ComponentSubDir: $(Build.Repository.Name)
-
-      - template: eng/pipelines/templates/upgrade-version.yml@ProjectSystemVSCodeExtensionRepo
-        parameters:
-          packageVersion: $(PackageVersion)
+        # https://devdiv.visualstudio.com/DevDiv/_library?itemType=VariableGroups&view=VariableGroupView&variableGroupId=434&path=DotNet-Project-System
+        # Variables used:
+        # - ApiScanConnectionString
+        - group: DotNet-Project-System
+      jobs:
+        - template: eng/pipelines/templates/build-official-release.yml@self
+          parameters:
+            SkipOptimize: ${{ parameters.SkipOptimize }}
+            CreateVSInsertion: ${{ parameters.CreateVSInsertion }}
+            InsertionVSBranch: ${{ parameters.InsertionVSBranch }}
+            CreateVSGreenInsertion: ${{ parameters.CreateVSGreenInsertion }}
+            InsertionVSGreenBranch: ${{ parameters.InsertionVSGreenBranch }}
+            SkipCompliance: ${{ parameters.SkipCompliance }}
```

Comment:
Remove

Reply:
should not be removed, per above comment